@nabilaalam95 Chittagong.md

```diff
@@ -0,0 +1,13 @@
+### Chittagong
+
+### Fun Facts
+  -Has a beach :D
+
+### Places to Eat
+  -Fried crabs in Naval
+
+### Places to Go
+    - Beach
+    -Butterfly Park
```

Comment:
It has some butterflies. But somehow they always hide from me :( @BinaryMuse 
![index](https://cloud.githubusercontent.com/assets/19973868/17125306/1dc8127c-5314-11e6-9873-da125d898b5a.jpg)